libs/community/langchain_community/chat_models/octoai.py

```diff
@@ -31,8 +31,8 @@ class ChatOctoAI(ChatOpenAI):
     """
 
     octoai_api_base: str = Field(default=DEFAULT_API_BASE)
-    octoai_api_token: SecretStr = Field(default=None)
-    model_name: str = Field(default=DEFAULT_MODEL)
+    octoai_api_token: SecretStr = Field(default=None, alias="octoai_api_token")
```

Comment:
```suggestion
    octoai_api_token: SecretStr = Field(default=None, alias="api_key")
```

see 
https://github.com/langchain-ai/langchain/issues/20085